### PR TITLE
feat(export): add checkbox to disable export of PII #141

### LINKED ISSUE #141: Add checkbox to disable export of PII
By default, `player.id`, `player.urlParams`, and `player. lastLogin.ip` should not be exported as they can contain Personal Identity Information (PII).

Tasks:
- [ ] add checkbox to export, that is always checked by default that states: "Remove Personal Identity Information (includes the player ID, URL parameters, and IP address)". This sets a remove_pii flag in the export request sent to server.
- [ ] In server export, if the remove_pii flag is set, remove the fields listed above from the export

### PR DESCRIPTION
**Pull request by hardisp**
## Description 
Add checkbox to let user remove Personal Identity Information (PII) when export data.

## Motivation and Context
#141 

## How Has This Been Tested?
This has been testing with export component.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: startup/server/export.js [hunk 1]
```diff
@@ -338,10 +338,11 @@ WebApp.connectHandlers.use("/admin/export", (req, res, next) => {
     gameDataFields
   );
 
-  const playerFields = [
-    "_id",
-    "id",
-    "urlParams",
+  const playerFields = ["_id"];
+  if (req.query.remove_pii === "false") {
```
Comment (npaton) [inline startup/server/export.js]: This should be the opposite, the default is false: 

```suggestion
  if (req.query.remove_pii !== "true") {
```

File: startup/server/export.js [hunk 2]
```diff
@@ -465,19 +467,21 @@ WebApp.connectHandlers.use("/admin/export", (req, res, next) => {
...
+      "roundId",
+      "stageId",
+      "name",
+      "jsonData",
+      "createdAt"
+    ];
+    saveFile("player-logs", playerLogFields, puts => {
+      batch(PlayerLogs)(p => puts(_.pick(p, playerLogFields)));
+    });
+  }
```
Comment (npaton) [inline startup/server/export.js]: Player logs should not be filtered out. They are game data, not PII (unless the experimenter did something wrong, which is beyond our reach for now) 👍

**Review by npaton — CHANGES_REQUESTED** [review]
Seeing this, I now think we should reverse the flag. First, double negations are unhelpful. And the default should be no PII if you pass no URL params, not the other way around.

We should call the flag `include_pii` and make it unselected by default in the UI. That should just be a quick reversal and a couple of copy changes:

"Add Personal Identity Information (includes the player ID, URL parameters, and IP address)"

File: startup/server/export.js (hunk 1 above)
Comment (npaton) [reply startup/server/export.js]: This now becomes:

```suggestion
  if (req.query.include_pii === "true") {
```

**Review by npaton — APPROVED** [review]

**Review by hubertusputu1 — APPROVED** [review]

**Pull request merged**